frontend/test/pytest/test_mitigation.py

```diff
@@ -122,13 +122,28 @@ def loop_1(i):  # pylint: disable=unused-argument
 
     @catalyst.qjit
     def mitigated_qnode(args, n):
-        return catalyst.mitigate_with_zne(
-            circuit, scale_factors=jax.numpy.array([1, 2, 3]), folding=folding
-        )(args, n)
+        return catalyst.mitigate_with_zne(circuit, scale_factors=[1, 3, 5, 7], folding=folding)(
```

Comment:
Is the only reason we are changing from numpy array to list is to have the concrete value?

Reply:
Yes for the check that is annoying to do at runtime

Reply:
If this is the case, during lowering (or the MLIR side), you could look up the definition of the operand. However, this only works if the constant is in the same function as the operation you are interested in.